src/webgpu/shader/execution/expression/binary/bitwise_shift.spec.ts

```diff
@@ -60,14 +61,12 @@ function isValidConstShiftRight(e1: ScalarValue, e2: number) {
   if (e2 >= bitwidth) {
     return false;
   }
-
   return true;
 }
 
 // Returns all cases of shifting e1 left by [0,63]. If `isConst` is true, cases that are
 // invalid for const eval are not returned.
-function generateShiftLeftCases(e1: ScalarValue, isConst: boolean): Case[] {
-  // This will need to change when AbstractInts are added in
+function generateShiftLeftConcreteCases(e1: ScalarValue, isConst: boolean): Case[] {
   assert(typeof e1.value === 'number');
 
   const bitwidth = e1.type.size * 8;
```

Comment:
nit: can we make the for loop in this function consistent with the one in `generateShiftLeftAbstractCases` : `for (let e2 = 0; e2 < 64; ++e2) {` ?

(GitHub is weird - I can't add comments on lines you did not modify)

Reply:
Done